File: findLetters.m
function[] = findLetters(centroids)
letterObjs = createLetterObjects();
[r c] = size(centroids);
text = '';
for i=1:r
    pointY = centroids(i).Centroid(2);
    pointX = centroids(i).Centroid(1);
    for j=1:23
        if(pointY >= (letterObjs(j).startPointY - 2)) && (pointY <= (letterObjs(j).endPointY + 2))
            text = strcat(text, letterObjs(j).letter);
            break;
        end
    end
end

disp(text);

File: Python/findLetters.py
def findLetters(imH, cnts):
	from imutils import contours
	import cv2
	from createLetters import createLetterObjs
	letters = createLetterObjs(imH)
	cnts = contours.sort_contours(cnts, method="left-to-right")[0]
	text = ""
	for c in cnts:
		(x, y, w, h) = cv2.boundingRect(c)
		pointY = y+h/2
		for l in letters:
			if pointY >= l.startPointY - 2 and pointY <= l.endPointY + 2:
				text = text  + l.letter
				break
	return text
